.jenkins/pytorch/macos-test.sh

```diff
@@ -4,22 +4,9 @@
 # shellcheck source=./macos-common.sh
 source "$(dirname "${BASH_SOURCE[0]}")/macos-common.sh"
 
-if [ -z "${CI}" ]; then
```

Comment:
These are legacy code.  Looking at the CI log, none of these are run anymore (python3.6):

* x86-64: https://github.com/pytorch/pytorch/actions/runs/3735754301/jobs/6339606870
* arm64: https://github.com/pytorch/pytorch/actions/runs/3735754301/jobs/6339527686